pkg/uroot/uroot.go

```diff
@@ -111,25 +111,21 @@ type Opts struct {
 	UseExistingInit bool
 }
 
-// ResolvePackagePaths takes a list of Go package import paths and directories
-// and turns them into exclusively import paths.
-//
-// Currently allowed formats:
-//   Go package imports; e.g. github.com/u-root/u-root/cmds/ls
-//   Paths to Go package directories; e.g. $GOPATH/src/github.com/u-root/u-root/cmds/ls
-//   Globs of paths to Go package directories; e.g. ./cmds/*
-func ResolvePackagePaths(env golang.Environ, pkgs []string) ([]string, error) {
-	var importPaths []string
+// Resolve package paths for one package import path string.
+func resolvePackagePath(env golang.Environ, pkg string) ([]string, error) {
+	// The empty string prefix is to search relative to the working directory
+	// TODO(subhav): Using env.SrcDirs includes the GOROOT. Dunno if that makes sense.
+	prefixes := append([]string{""}, env.SrcDirs()...)
```

Comment:
maybe you can use "." to be more explicit about the cwd?

Reply:
Actually, this breaks absolute paths. Leaving it the way it is, and added a test for that.